lib/util/exec/index.ts

```diff
@@ -79,7 +80,8 @@ function getRawExecOptions(opts: ExecOptions): RawExecOptions {
     cwd,
   };
   // Set default timeout to 15 minutes
-  rawExecOptions.timeout = rawExecOptions.timeout || 15 * 60 * 1000;
+  rawExecOptions.timeout =
+    rawExecOptions.timeout ?? defaultExecutionTimeout ?? 15 * 60 * 1000;
```

Comment:
Isn't this defaultExecutionTimeout in minutes, not milliseconds?

Reply:
It sounds you are right, I'll fix this